spec/routing/authn_routing_spec.rb

```diff
@@ -12,7 +12,24 @@
       id: 'kevin.gilpin@inscitiv.com'
     )
   end
-  
+
+  it "routes POST /authn-oidc/the-service/the-account/authenticate to authenticate#authenticate_oidc" do
+    expect(post: '/authn-oidc/the-service/the-account/authenticate').to route_to(
+      controller: 'authenticate',
+      action: 'authenticate_oidc',
+      service_id: 'the-service',
+      account: 'the-account'
+    )
+  end
+
+  it "routes POST /authn-oidc/the-account/authenticate to authenticate#authenticate_oidc" do
+    expect(post: '/authn-oidc/the-account/authenticate').to route_to(
+      controller: 'authenticate',
+      action: 'authenticate_oidc',
+      account: 'the-account'
+    )
+  end
+
```

Comment:
I don't think there's much benefit to these tests.  It's almost testing the rails routing framework itself, and besides, what they're resting will be implicitly tested by our end to end tests.

I would remove them.

Reply:
These tests aren't part of our PR (we just added some tests) so I'd rather not get this decision as part of the review